### ISSUE TITLE
Hide fields for guest users

### ISSUE THREAD

**Issue by indrabhusanroy** [YetiForceCompany/YetiForceCRM#9171]
Hi, Is there a way to hide this fields from a guest user. The fields are:

- Modified time
- Record number
- Created time
Please guide me how to hide this fields from a guest users.

Thanks...

**Comment by vovpff** [comment]
@indrabhusanroy try profiles

**Comment by indrabhusanroy** [comment]
Yes I have tried that but of no use as fields: Modified time, Record Number & Created Time cannot be touched. They cannot be made Invisible/read Only/write as other custom fields can be done.

**Comment by vovpff** [comment]
in this case you can change this params manually in database but his is hardcore
